.gitignore

```diff
@@ -1,8 +1,9 @@
 # .DS_Store files!
 .DS_Store
 
-# don't track .pyc files
+# don't track .pyc files or autopkgc
 *.pyc
+autopkgc
```

Comment:
This is autogenerated when importing autopkg directly via `imp.load_source()` for unit testing.